config/default-broker-channel.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 The Knative Authors
```

Comment:
I re-arranged the config folders a bit to match eventing and serving... 
this should go inside config/core/configmaps. Let's rename it to be `config-br-default-channel.yaml`

And then you create a symlink to it from config. Call it `400-config-br-default-channel.yaml`

Reply:
Done